base/util_testing.go

```diff
@@ -670,3 +670,33 @@ func ForAllDataStores(t *testing.T, testCallback func(*testing.T, sgbucket.DataS
 		})
 	}
 }
+
+// clusterCompatibility = MajorVersion * 65536 + MinorVersion
+const minCompatVersionForCollections = 7 * 65536
+
+// map of server names to cluster compat versions (string->int)
+var serverClusterCompat sync.Map
```

Comment:
I'm not sure I understand the intention of the `c.server` lookup? There's only one server behind a test bucket pool